stellarphot/visualization/comparison_functions.py

```diff
@@ -707,11 +717,23 @@ def save_tess_files(self, button=None):
         """
         if self._field_name.value:
             self.tess_field_view()
-            self.iw.save(self._field_name.value, overwrite=True)
+            # Remove output file if it exists
+            if os.path.exists(self._field_name.value) and self.overwrite_outputs:
```

Comment:
It would be nice to switch this to `pathlib` instead. I can do that later in a separate PR if you prefer,

Reply:
I'll get this addressed shortly.